src/modules/services/submodule/assets/views/ViewServicesAssetsModifySupplyChange.vue

```diff
@@ -111,70 +146,101 @@ export default {
+    const isMultiSigBool = ref(isMultiSig(walletState.currentLoggedInWallet.selectDefaultAccount().address));
 
     const showNoBalance = ref(false);
-    // if(balance.value < rentalFee.value){
-    //   showNoBalance.value = true;
-    //   disabledPassword.value = true;
-    //   disabledSupply.value = true;
-    //   disabledClear.value = true;
-    // }else{
-    //   disabledPassword.value = false;
-    //   disabledSupply.value = false;
-    //   disabledClear.value = false;
-    // }
-
-    const supply = ref(0);
-    const accounts = computed( () => appStore.getWalletByName(appStore.state.currentLoggedInWallet.name).accounts);
-    const moreThanOneAccount = computed(()=> (appStore.getWalletByName(appStore.state.currentLoggedInWallet.name).accounts.length > 1)?true:false);
+    const showNoAsset = ref(false);
+    const isNotCosigner = computed(() => getMultiSigCosigner.value.list.length == 0 && isMultiSig(selectedAccAdd.value) && !showNoAsset.value);
+
+    const supply = ref('0');
+    const accounts = computed( () => walletState.currentLoggedInWallet.accounts);
+    const moreThanOneAccount = computed(()=> (walletState.currentLoggedInWallet.accounts.length > 1)?true:false);
+    const transactionFee = ref('0.000000');
+    const transactionFeeExact = ref(0);
+
+    const ownerPublicAccount = ref(WalletUtils.createPublicAccount(walletState.currentLoggedInWallet.selectDefaultAccount().publicKey, networkState.currentNetworkProfile.network.type));
+
+    const getMultiSigCosigner = computed(() => {
+      return AssetsUtils.getCosignerList(selectedAccAdd.value);
+    });
 
     const changeSelection = (i) => {
+      assetOption.value.clear();
       selectedAccName.value = i.name;
       selectedAccAdd.value = i.address;
       balance.value = i.balance;
-      // (balance.value < rentalFee.value)?showNoBalance.value = true:showNoBalance.value = false;
       showMenu.value = !showMenu.value;
       currentSelectedName.value = i.name;
       isMultiSigBool.value = isMultiSig(i.address);
+      ownerPublicAccount.value = WalletUtils.createPublicAccount(i.publicKey, networkState.currentNetworkProfile.network.type);
+      showNoAsset.value = (assetOptions.value.length == 0)?true:false;
+      showNoBalance.value = ((balance.value < rentalFee.value) && !isNotCosigner.value && !showNoAsset.value)?true:false;
     }
 
-    const assetOptions = () => {
-      let asset = [];
-      return asset;
+    const assetSupply = ref(0);
+    const assetAmount = ref(0);
+    const assetDuration = ref('0 Day');
+    const assetDivisibility = ref(0);
+    const assetTransferable = ref(false);
+    const assetMutable = ref(false);
+
+    const changeAsset = (assetId) => {
+      const selectedAsset = walletState.currentLoggedInWallet.selectDefaultAccount().assets.find((asset) => asset.idHex === assetId);
+      assetSupply.value =Helper.amountFormatterSimple(selectedAsset.amount, selectedAsset.divisibility);
```

Comment:
variable is assetSupply, but you use the asset amount value

Reply:
Thanks, have corrected the property